engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -423,7 +410,15 @@ object EngineJobExecutionActor {
 
   private[execution] case class ResponsePendingData(jobDescriptor: BackendJobDescriptor,
                                                     bjeaProps: Props,
-                                                    hashes: Option[Try[CallCacheHashes]] = None) extends EJEAData
+                                                    hashes: Option[Try[CallCacheHashes]] = None) extends EJEAData {
+
+    def withResponse(success: SucceededResponse) = SucceededResponseData(success, hashes)
+
+    def withResponse(response: BackendJobExecutionResponse) = response match {
+      case success: SucceededResponse => SucceededResponseData(success, hashes)
```

Comment:
I don't think this first case can match, the more derived version above would hit first